### PR TITLE
fix markdown syntax error

### PR DESCRIPTION
**Pull request by xuwei-k**

### CODE REVIEW

File: docs/manual/working/scalaGuide/main/sql/slick/PlaySlickMigrationGuide.md [hunk 1]
```diff
@@ -119,4 +119,4 @@ For a full example, have a look at [this sample projet](https://github.com/playf
...
 
-The consequence of the above is that you are in charge of creating and managing your project's database schema. Therefore, whenever you make a change a Slick table in the code, make sure to also update the database schema. If you find it tedious to manually keep in sync your database schema and the related table definition in your code, you may want to have a look at the code generation feature available in Slick.
\ No newline at end of file
+The consequence of the above is that you are in charge of creating and managing your project's database schema. Therefore, whenever you make a change a Slick table in the code, make sure to also update the database schema. If you find it tedious to manually keep in sync your database schema and the related table definition in your code, you may want to have a look at the code generation feature available in Slick.
```
Comment (dotta) [inline docs/manual/working/scalaGuide/main/sql/slick/PlaySlickMigrationGuide.md]: Was there any change here? I can't find it :-)

**Comment by dotta** [comment]
@xuwei-k Thanks for the PR. Happy to merge as soon as my above comment is cleared out :-)

File: docs/manual/working/scalaGuide/main/sql/slick/PlaySlickMigrationGuide.md (hunk 1 above)
Comment (xuwei-k) [inline docs/manual/working/scalaGuide/main/sql/slick/PlaySlickMigrationGuide.md]: Ah sorry, there is no changes. maybe inserted new line by my editor.

Comment (xuwei-k) [inline docs/manual/working/scalaGuide/main/sql/slick/PlaySlickMigrationGuide.md]: fixed

**Comment by dotta** [comment]
Thanks again!

**Pull request merged**